File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/C++/s466963472_comments_removed.cpp
#include <iostream>
#include <limits>

using namespace std;

static int d;

static bool read() {
    if (!(cin >> d)) {
        cin.clear();
        cin.ignore(numeric_limits<streamsize>::max(), '\n');
        return false;
    }
    return true;
}

static int solve() {
    int ans = 0;
    for (int i = 1; i * d < 600; i++) {
        int x = i * d;
        int y = x * x;
        ans += y * d;
    }
    return ans;
}

int main() {
    while (read()) {
        cout << solve() << endl;
    }
    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/Go/s466963472_comments_removed.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

var d int

func read() bool {
	var input string
	_, err := fmt.Scanln(&input)
	if err != nil {
		return false
	}
	if _, err := strconv.Atoi(input); err != nil {
		return false
	}
	d, _ = strconv.Atoi(input)
	return true
}

func solve() int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	for read() {
		fmt.Println(solve())
	}
}

// End of Code
